src/api/core/tokenManager.ts

```diff
@@ -149,24 +158,31 @@ export class TokenManager {
 
   /**
    * Add new token pair and run refresh digest
-   * @param {string} auth Authenticate alias
+   * @param {Array<string | undefined>} aliases an array of authenticate aliases
    * @param {Tokens} tokens Token pair
    * @param {boolean} defaults Whether to use this token by default
    */
-  public addTokens(auth: string, tokens: Tokens, defaults?: boolean) {
-    this.storage.setTokens(auth, tokens, defaults);
-    this.startRefreshDigest(auth);
+  public addTokens(aliases: Array<string | undefined>, tokens: Tokens, defaults?: boolean) {
```

Comment:
is there any special reason `undefined` can come from here? 🤔

Reply:
yes, since our customer provides this interface:
```
export interface IUMSSignInOptions {
  email: string;
  password: string;
  default?: boolean;
  alias?: string;
}
```
and then signIn() function does this:
```
this.tokenManager.addTokens([user.email, user.alias], tokens, user.default);
```
and from here `user.alias` can be undefined :)

Reply:
ok, sounds reasonable!